src/backend/distributed/operations/node_protocol.c

```diff
@@ -798,6 +798,8 @@ void
 GatherIndexAndConstraintDefinitionList(Form_pg_index indexForm, List **indexDDLEventList,
 									   int indexFlags)
 {
+	PushOverrideEmptySearchPath(CurrentMemoryContext);
```

Comment:
comment on why we need this could be nice, so we don't accidentally delete it again :)